addons/sales_team/models/crm_team.py

```diff
@@ -146,17 +146,28 @@ def _graph_data(self, start_date, end_date):
                       AND DATE(%(date_column)s) <= %(end_date)s
                       %(extra_conditions)s
                     GROUP BY x_value;"""
+
+        # apply rules
+        graph_sql_table = self._graph_sql_table()
+        extra_conditions = self._extra_sql_conditions()
+        GraphModel = self.env[".".join(graph_sql_table.split("_"))]
```

Comment:
There's no way to obtain the python model from the sql table. This work-around will have to do, or we have to change the _graph_sql_table() to give the model instead of the table.